.github/workflows/update-nixpkgs.yaml

```diff
@@ -0,0 +1,41 @@
+name: update-nixpkgs
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: "5 8 * * *"
```

Comment:
I think it would be great if we already had the result in our Hydra in the morning, so maybe 3am?